Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -134,6 +136,25 @@ final class PledgeAmountCell: UITableViewCell, ValueCell {
   }
 }
 
+extension PledgeAmountCell: UITextFieldDelegate {
+  func textField(
+    _ textField: UITextField, shouldChangeCharactersIn _: NSRange, replacementString string: String
+  ) -> Bool {
+    let decimalSeparatorCharacters = CharacterSet.ksr_decimalSeparators()
+    let existingCharacters = CharacterSet(charactersIn: textField.text ?? String.init())
```

Comment:
`String.init()` -> `""`?

Reply:
LOL..exactly what I had...then I was like...no Justin will prefer `String.init()` 🤦‍♂

Reply:
😂 we actually have a helper that I like called `coalesceWith` which is just `??` under the hood. but it would read `textField.text.coalesceWith("")`, I'm cool with either.

Reply:
That's good to know...I can use that one...but it's not something we do consistently